### DISCUSSION TITLE
User case of git-cliff

### DISCUSSION THREAD

**Discussion by tisonkun** [orhun/git-cliff#584] [General]
I'm starting a discussion for using git-cliff to generate release notes for GreptimeDB - https://github.com/GreptimeTeam/greptimedb/issues/3609

Just for your information. I found other projects like aide has a `cliff.toml` file so I feel more confident to leverage this tool.

**Comment by orhun** [comment]
Awesome! Just let me know if you need any help integrating it.

**Comment by orhun** [comment]
I think we can now close this. LMK if there is anything!

**Reply by tisonkun** [reply]
Sure. Thank you! 

Maybe you'd take a look at https://github.com/orhun/git-cliff/pull/599.
